user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/RegistrationValidationExecutor.java

```diff
@@ -0,0 +1,45 @@
+package life.qbic.datamanager.views.projects.project.measurements;
+
+import java.util.Collection;
+import java.util.Objects;
+import java.util.Optional;
+import life.qbic.projectmanagement.application.measurement.NGSMeasurementMetadata;
+import life.qbic.projectmanagement.application.measurement.ProteomicsMeasurementMetadata;
+import life.qbic.projectmanagement.application.measurement.validation.ValidationResult;
+import life.qbic.projectmanagement.application.measurement.validation.ValidationService;
+import life.qbic.projectmanagement.application.measurement.validation.ValidationService.Domain;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * <b>Registration Validation Executor</b>
+ * <p>
+ * Implementation of the {@link ValidationExecutor} interface,
+ * handling the validation of newly registered {@link life.qbic.projectmanagement.application.measurement.MeasurementMetadata}
+ *
+ */
+@Component
+public class RegistrationValidationExecutor implements ValidationExecutor {
```

Comment:
Weird naming. Why not `MeasurementMetadataValidator`?
I think this is not for every registration but only for measurement metadata? What is the difference between this executor and the validation service?

Reply:
The naming is related to when this executor is used. This PR introduces different validation triggering executor for "Registration" and "Editing" measurement metadata. 
I'll happily adapt the naming scheme, but then this should also be done for ValidationResult Validation Report Validation Executor etc. to be consistent 🤔

Reply:
Ah so the executor runs the validation service based on a trigger? Then `MeasurementRegistrationTriggeredValidator` would fit better. I do not see why this changes anything for `ValidationResult` and `ValidationReport`. The interface name can stay the same IMHO.